README.md

```diff
@@ -40,6 +40,11 @@ npm install chrome-launcher
   // Do note, many flags are set by default: https://github.com/GoogleChrome/chrome-launcher/blob/master/src/flags.ts
   chromeFlags: Array<string>;
 
+  // (optional) Additional preferences to be set in Chrome, for example: {'download.default_directory': __dirname}
+  // See: https://src.chromium.org/viewvc/chrome/trunk/src/chrome/common/pref_names.cc?view=markup
```

Comment:
viewvc links are some oldddd version of the chromium repo. you can see a 2014 date up there.

here's the modern equivalent of that page: https://chromium.googlesource.com/chromium/src/+/main/chrome/common/pref_names.cc

via the "view in gittiles" _Link_ from here: https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/pref_names.cc


i'll change that and merge this in.

Reply:
```suggestion
  // See: https://chromium.googlesource.com/chromium/src/+/main/chrome/common/pref_names.cc
```

Reply:
Thanks Paul!